testing/kuttl/e2e/cluster-pause/01--cluster-paused.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: cluster-pause
+spec:
+  postgresVersion: ${KUTTL_PG_VERSION}
+  paused: true
+  instances:
+    - name: instance1
+      replicas: 2
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: 1Gi
+  backups:
+    pgbackrest:
+      repos:
+      - name: repo1
+        volume:
+          volumeClaimSpec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: 1Gi
```

Comment:
If we're just updating `spec.paused` and the replicas, do we need the whole cluster info here? Probably can drop out the `backups` section, but maybe not the `instances` (since we're updating the object in that array -- which took me a moment to recognize).

Reply:
Good call, testing showed this can definitely be slimmed down a bit, so I made that change and added a comment per above.